combat-trainer.lic

```diff
@@ -566,7 +566,11 @@ class SpellProcess
     echo("  @offensive_spell_cycle: #{@offensive_spell_cycle}") if $debug_mode_ct
 
     @spell_data = get_data('spells').spell_data
-
+    @offensive_spells.each.with_index do |spell, i| 
+      if @spell_data[spell['name']]
+        @offensive_spells[i] = spell.merge(@spell_data[spell['name']]) 
```

Comment:
Note to self: I need to swap the things being merged... If there are duplicate values then they are retained from y in `x.merge(y)` So we should keep the user settings